EmbeddedSocialClient/sdk/src/main/java/com/microsoft/embeddedsocial/image/ImageLoader.java

```diff
@@ -61,22 +61,22 @@ public void onImageLoadFailed(Picasso picasso, Uri uri, Exception exception) {
     }
 
     public static void load(ImageView imageView, String uri) {
-        performAction(() -> picasso.load(uri).into(imageView));
+        performAction(() -> picasso.load(uri).placeholder(R.drawable.placeholder).into(imageView));
```

Comment:
R is not included in the imports

Reply:
Ok I will make the required the changes.